mods/default/nodes.lua

```diff
@@ -2868,6 +2868,18 @@ minetest.register_node("default:cloud", {
 	groups = {not_in_creative_inventory = 1},
 })
 
+minetest.register_node("default:barrier", {
+	description = S("Barrier"),
+	tiles = {"default_barrier.png"},
+	inventory_image = "default_cloud.png",
+	is_ground_content = false,
+	sounds = default.node_sound_defaults(),
```

Comment:
It shouldn'tmake any sound, for example when walked on.

Reply:
Done!

Reply:
Doesn't seem done.

Reply:
Oops I forgot to push it xD
I've done it now.